sagemaker-training-compiler/huggingface/pytorch_single_gpu_single_node/vision-transformer/vision-transformer.ipynb

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        pytorch_version="1.10.2",
</pre><p>The version we are testing is 1.11, which is as yet unreleased:</p><p>&nbsp;&nbsp;&nbsp;image_uri="669063966089.dkr.ecr.us-west-2.amazonaws.com/pr-huggingface-pytorch-training:1.11.0-transformers4.21.1-gpu-py38-cu113-ubuntu20.04-pr-1824-2022-08-08-10-57-02",</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>